executor/analyze.go

```diff
@@ -1321,12 +1346,21 @@ func (e *AnalyzeColumnsExec) subMergeWorker(resultCh chan<- *samplingMergeResult
 			resultCh <- &samplingMergeResult{err: err}
 			return
 		}
+		colRespSize := colResp.Size()
+		memTracker.Consume(int64(colRespSize))
 		subCollector := statistics.NewRowSampleCollector(int(e.analyzePB.ColReq.SampleSize), e.analyzePB.ColReq.GetSampleRate(), l)
 		subCollector.Base().FromProto(colResp.RowCollector)
+		subCollectorSize := subCollector.Base().MemoryUsage()
```

Comment:
The sub is merged. Do we need to subtract its usage after it's merged?

Reply:
Yes, lastToReleaseMem contains it.